### DISCUSSION TITLE
Need license information for this tool to be used by 90k+ users for a company

### DISCUSSION THREAD

**Discussion by PallaviXP** [requarks/wiki#5676] [General]
wiki.js :- Need license information for this tool to be used by 90k+ users for a company,
on-premise and cloud need both solutions information.

**Comment by NGPixel** [comment]
Could you clarify your question? What info do you need exactly?

Wiki.js is licensed under AGPLv3. You can install it anywhere, whether its cloud or on-promise.

**Reply by vrao13** [reply]
We need the following information:
1. Is wiki.js a free tool or there is licensing cost involved?
2. Will it support 90K users? 
3. Will it support migration from Confluence wiki? 
4. Can we setup a demo to understand it better before making the final call? 
5. Do you have a dedicated support contact, a CSM maybe to help us with our queries?
6. Are trainings available?

**Reply by NGPixel** [reply]
1. Wiki.js is open source and free.
2. Yes.
3. You can export confluence to markdown / html files and import them into wiki.js.
4. You can install it on any machine for testing. Or use a prebuilt DigitalOcean droplet. Or check hostwiki.com for hosted demos.
5. No but some 3rd party services do. See stellarhosted.com or hostwiki.com for example.
6. No.

**Reply by vrao13** [reply]
Appreciate your quick response. We will try these options.

**Reply by jkour** [reply]
@vrao13 About #3, check this https://apps.kouraklis.com/conflux/
